common/types/object.go

```diff
@@ -54,6 +54,10 @@ func (o *protoObj) ConvertToNative(typeDesc reflect.Type) (interface{}, error) {
 	if typeDesc == anyValueType {
 		return ptypes.MarshalAny(o.Value().(proto.Message))
 	}
+	// If the duration is already assignable to the desired type return it.
```

Comment:
Do you mean "If the object is already assignable to the desired type return it" ?